assets/js/util.ts

```diff
@@ -178,22 +178,22 @@ export const placesWithSelectedAlert = (
     : [];
 };
 
-export const sortScreens = (screenList: Screen[]) => {
-  const screenTypeOrder = [
-    "dup",
-    "dup_v2",
-    "bus_shelter_v2",
-    "bus_eink",
-    "bus_eink_v2",
-    "gl_eink_single",
-    "gl_eink_double",
-    "gl_eink_v2",
-    "pre_fare_v2",
-    "triptych_v2",
-    "solari",
-    "pa_ess",
-  ];
+const screenTypeOrder = [
+  "dup",
+  "dup_v2",
+  "bus_shelter_v2",
+  "bus_eink",
+  "bus_eink_v2",
+  "gl_eink_single",
+  "gl_eink_double",
+  "gl_eink_v2",
+  "pre_fare_v2",
+  "triptych_v2",
+  "solari",
```

Comment:
Looks like "busway_v2" still needs to be added here.

Reply:
oops, good catch!